pandas-stubs/core/series.pyi

```diff
@@ -371,6 +374,10 @@ class Series(IndexOpsMixin[S1], NDFrame):
     @overload
     def __getitem__(self, idx: Scalar) -> S1: ...
     def __setitem__(self, key, value) -> None: ...
+    @overload
+    def get(self, key: Hashable, default: None = ...) -> S1 | None: ...
+    @overload
+    def get(self, key: Hashable, default: S1 | _T = ...) -> S1 | _T: ...
```

Comment:
Remove the `...` for `default` .  Also, split this into 2 overloads, one for `S1` and another for `_T` (which should change to `T`) .  That might require an `ignore` because of overlaps.